discipl/Issues/demoVIL/description.xml

```diff
@@ -0,0 +1,94 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Virtueel Inkomstenloket</en>
+		<nl>Demo Virtueel Inkomstenloket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo VIL</en>
+		<nl>Demo VIL</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>discipl</IssuerID>
+	<CredentialID>demoVIL</CredentialID>
+	<Description>
+		<en>Demo Virtueel Inkomstenloket (VIL) using Rule Services</en>
+		<nl>Demo Virtueel Inkomstenloket (VIL) die gebruik maakt van Regel Services</nl>
+	</Description>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.discipl.demoVIL.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.discipl.demoVIL.html</nl>
+	</IssueURL>
+
+	<Attributes>
+		<Attribute id="Woonplaats">
```

Comment:
Nitpick: this is the only attribute identifier starting with an uppercase letter.